flood_adapt/object_model/interface/site.py

```diff
@@ -201,6 +201,7 @@ class FiatModel(BaseModel):
     new_development_file_name: Optional[str] = None
     save_simulation: Optional[bool] = False
     svi: Optional[SVIModel] = None
+    infographics: Optional[bool] = True
```

Comment:
Should the default be True?

Reply:
that is a good point. Probably it makes more sense for False.